CHANGELOG-1.3.md

```diff
@@ -1,9 +0,0 @@
-# Changelog since v1.2.0
-
-## Breaking Changes
-
-- Add a prefix "external-snapshotter-leader" and the driver name to the snapshotter leader election lock name. Rolling update will not work if leader election is enabled because the lock name is changed in v1.3.0. ([#129](https://github.com/kubernetes-csi/external-snapshotter/pull/129), [@zhucan](https://github.com/zhucan))
```

Comment:
Are these changes still new for 2.0?

Reply:
Good point.  This one is not backported.